### DISCUSSION TITLE
Move the search bar?

### DISCUSSION THREAD

**Discussion by mgriff88** [squidfunk/mkdocs-material#7189] [Q&A]
Hi all. Does anyone know if it's possible to move the search bar elsewhere? In my use case, I'd like to create a landing page with the search bar right in the center, like so:

> **What are you looking for today?**
> ( search bar here )

See Slack's Help site for a great example. https://slack.com/intl/en-ie/help

My thinking is, I'd like the landing page to have this unique placement of the search bar, and then every internal docs page will have it back in the header as expected.

I've tried putting the search block into the content area of the landing page template, but it didn't quite work... when you clicked in the search bar, the actual search panel still popped up in the upper right hand corner, so it was an odd experience.

Thanks, appreciate the help!

**Comment by squidfunk** [comment]
Thanks for asking! It's currently not easy to do this, but we're planning to move to a more modular architecture in the future, which should make it simpler. The use case you describe is definitely sound, thanks for sharing!

**Reply by mgriff88** [reply]
Thank you for the update!

**Reply by jipelski** [reply]
I'm also trying to create a landing page with the search bar right in the center. I have encountered the same odd experience as @mgriff88 when moving the search bar. Is there any update on this?

**Reply by squidfunk** [reply]
No update yet, but we've moved past planning and are now working on a modular architecture. No ETA yet, though, and please keep in mind that we are a very small team, so it will take a little more time.
